### PR TITLE
v2

### PR DESCRIPTION
**Pull request by i-gao**

### CODE REVIEW

File: open_flamingo/eval/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# OpenFlamingo Evaluation Suite
```
Comment (anas-awadalla) [inline open_flamingo/eval/README.md:1]: Update instructions and add links to files for each benchmark.

File: open_flamingo/train/train_utils.py [hunk 2]
```diff
@@ -149,52 +161,76 @@ def train_one_epoch(
...
+            )
+            if args.fsdp:
+                model.lang_encoder.get_input_embeddings().weight.grad = (
+                    embed_grad * zero_mask
+                )
+            else:
+                model.module.lang_encoder.get_input_embeddings().weight.grad = (
+                    embed_grad * zero_mask
+                )
+            print(
```
Comment (anas-awadalla) [inline open_flamingo/train/train_utils.py:199]: remove embed masking prints

File: open_flamingo/src/factory.py [hunk 3]
```diff
@@ -14,6 +14,7 @@ def create_model_and_transforms(
     cross_attn_every_n_layers: int = 1,
     use_local_files: bool = False,
     decoder_layers_attr_name: str = None,
+    freeze_lm_embeddings: bool = False,
```
Comment (anas-awadalla) [inline open_flamingo/src/factory.py:17]: maybe we should print a warning if we are doing this? I think it would reduce perf.

**Comment by i-gao** [comment]
Merged by command line

**Pull request closed without merging**